mflowgen/full_chip/construct.py

```diff
@@ -456,6 +458,12 @@ def construct():
 
   # Antenna DRC node needs to use antenna rule deck
   antenna_drc.update_params( { 'drc_rule_deck': parameters['antenna_drc_rule_deck'] } )
+
+  # Remove unresolved reference assertion from DC because dragonphy is an unresolved reference
+  dc_postconditions = dc.get_postconditions()
+  dc_postconditions.remove( "assert 'Unresolved references' not in File( 'logs/dc.log' )" )
+  dc_postconditions.remove( "assert 'Unable to resolve' not in File( 'logs/dc.log' )" )
```

Comment:
I agree that we can remove the "unresolved references" condition, because I think it's probably 100% redundant with the "unable to resolve" condition.

But I don't think we have to get rid of that second condition entirely, do we? I looked at the synthesis log for an old run (build 1981 ~ Aug 2) and only found these warnings that would fit:
```
gold.1981/full_chip/18-synopsys-dc-synthesis/mflowgen-run.log:
Warning: Unable to resolve reference 'PCORNER' 
Warning: Unable to resolve reference 'dragonphy_top' 
```

I think we fixed PCORNER separately, yes? So all we really need is a condition that *allows* `"Unable to resolve reference 'dragonphy_top'"` and disallows all other `"Unable to resolve"` messages. off the top of my head I don't know the mflowgen syntax for that, maybe @ctorng would know.

Anyway: I'ma go ahead and approve without this change, so you can decide for yourself what you want to do.

Reply:
Well there's no doubt that both assertions have to be removed for this to work. But the second could be replaced with something else.

Reply:
Rather than specifically target the dragonphy_top unresolved reference, I added an assertion that there is only 1 unresolved reference. It's possible to do what you said, but it requires more work, and I think this is sufficient.